content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -12,7 +12,7 @@ min-kubernetes-server-version: 1.16
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.16" state="beta" >}}
+{{< feature-state for_k8s_version="v1.22" state="ga" >}}
```

Comment:
Netlify build error:
https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/610d44bd89d3c60007aee9a9
> "alpha, beta, deprecated, stable" is not a valid feature-state, use one of %!q(MISSING)

It seems that `"ga"` should be one of: `"alpha, beta, deprecated, stable"`.

Reply:
Appreciate it, done.